### ISSUE TITLE
functional and modern harmonious vehicle popup

### ISSUE THREAD

**Issue by EdilonCarvalho** [traccar/traccar#4879]
The vehicle popup is good at first but if we add more attributes it gets much bigger obstructing the view of the map and vehicles.
A horizontal popup like the link image will be much better, allowing you to place more attributes without compromising visibility in addition to giving the new interface a more modern look, as this is the proposal.

https://uploaddeimagens.com.br/imagens/wl0ZaEo

![Print platform popup](https://user-images.githubusercontent.com/108483132/176752011-61361392-bf92-41a2-87f5-cc3c4a2f8a59.png)

**Comment by tananaev** [comment]
Doesn't this only solve problem when attribute is too long and gets wrapped on the next line? Like address, for example.

**Comment by tananaev** [comment]
Re-positioned? You mean multiple columns?

**Comment by EdilonCarvalho** [comment]
focusing only on the popup, it would be something like this but it doesn't need to support as many attributes but a larger amount than the standard format without compromising the visibility of the map or attributes

![platform](https://user-images.githubusercontent.com/108483132/176755541-05b7fa17-7da7-4854-b092-cd9436fb6560.png)

**Comment by EdilonCarvalho** [comment]
I just see that it would be much more modern and functional by simplifying the immediate visualization of what is needed, both on the map and some more attributes to be added 

but ok I will try to implement this for my use

**Comment by tananaev** [comment]
There's always a balance between simplicity and displaying information. You can't say that displaying everything is better in every single case. Anyway, let's see what other think. Maybe it make sense to split the popup at least into 2 columns.

**Comment by EdilonCarvalho** [comment]
yes, it could be one just for the address and one or two more for other information

Thank you for your attention Anton

**Comment by apnagaev** [comment]
Hi!
@EdilonCarvalho  is right. You can track device only if device selected, but it if selected - you can't view center of screen, you can't view live routs becouse all rout behind popup.
At mobile view popup occupy all screen, but you can't track device if deselect device (closed popup).
Need change interface how offer @EdilonCarvalho  or implement minimaze for popup or/and implement moving popup on screen
